autoscheduler/frontend/src/components/SchedulingPage/SchedulePreview/SchedulePreview.tsx

```diff
@@ -83,7 +87,7 @@ const SchedulePreview: React.FC = () => {
         <div id={styles.cardHeader}>Schedules</div>
       }
     >
-      <List className={styles.list}>
+      <List className={styles.list} disablePadding>
```

Comment:
Not really related to this PR but the padding on the top and bottom of the list bugged me so I changed it, here's before:
![image](https://user-images.githubusercontent.com/28655462/86315919-190cc080-bbf1-11ea-83b0-088ab47b14a6.png)
and after:
![image](https://user-images.githubusercontent.com/28655462/86315935-2164fb80-bbf1-11ea-9d0d-7d1cbee4f4c5.png)
Let me know if you guys don't like this change, but I think it looks nicer